exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
remember this exercise is about comparison operators. Can you use the two variables `studentCount` and `mentorCount` with a comparison operator so that the expression evaluates to `true`?
These are the possible comparison operators
```
>   greater than
<   less than
<=  less than or equal
>=  greater than or equal
=== same value
!== not the same value
```

Reply:
I made the changes. Thanks.